portfolio.html

```diff
@@ -0,0 +1,100 @@
+		<div class="project-item">
+			<img class="project-image" src="nova2.jpeg" alt="Adorable puppy in curtains"/>
+			<div class="project-info">
+				<h3>Project One</h3>
+				<div class="project-description">In my first project I created a small little floof called Nova and she was perfect in everyway. She has little paws and soft fur and the best grin, and she raised my level of happiness by about 1000000%. In my first project I created a small little floof called Nova and she was perfect in everyway. She has little paws and soft fur and the best grin, and she raised my level of happiness by about 1000000%. In my first project I created a small little floof called Nova and she was perfect in everyway. She has little paws and soft fur and the best grin, and she raised my level of happiness by about 1000000%. In my first project I created a small little floof called Nova and she was perfect in everyway. She has little paws and soft fur and the best grin, and she raised my level of happiness by about 1000000%.</div>
+				<div>
+					<a href="#">GitHub Link</a>
+					<a href="#">View project</a>
+				</div>
+			</div>
+		</div>
+		<div class="project-item">
+			<img class="project-image" src="nova3.png" alt="Nova with slipper"/>
+			<div class="project-info">
+				<h3>Project Two</h3>
+				<div class="project-description">This is the description of the second project</div>
+				<div>
+					<a href="#">GitHub Link</a>
+					<a href="#">View project</a>
+				</div>
+			</div>
+		</div>
+		<div class="project-item">
+			<img class="project-image" src="Nova1.jpeg" alt="black and white Nova" />
+			<div class="project-info">
+				<h3>Project Three</h3>
+				<div class="project-description">This is the description of the third project</div>
+				<div>
+					<a href="#">GitHub Link</a>
+					<a href="#">View project</a>
+				</div>
+			</div>
+		</div>
+	</section>
+</div>
+	<!-- about me -->
+<div>
+	<section id="about">
+		<h2>About Me</h2>
+		<div class="about">
+			<p>Hi, I'm Ellie and I like coding, my dog Nova and gaming. I'm currently completing iO Academy's Full Stack Track programming bootcamp (and loving it!).</p> 
+			<p>In my free time, I enjoy learning Japanese, walking my dog, playing piano, spending far too much time on Zelda and finding new hobbies.</p>
+			<p>If you'd like to know more about my professional history, please visit my <a href="timeline.html" target="_blank">timeline!</a></p>
+		</div>
+	</section>
+</div>
+	<!-- contact -->
+<div>
+	<section id="contact">
+		<h2>Contact Me</h2>
+		<div class="media-links">
+			<a href="https://www.linkedin.com/in/eleanor-roberts-80117a16a/" class="fa fa-linkedin"></a>
+			<a href="https://www.freecodecamp.org/elmon" class="fa fa-free-code-camp"></a>
+			<a href="mailto: eleanorlroberts@googlemail.com" class="fa fa-envelope-square"></a>
+			<a href="https://github.com/EleanorRoberts" class="fa fa-github-square"></a>
+		</div>
+	</section>
+</div>
+	<!-- footer -->
+<footer>
+	<section>
```

Comment:
Could just use a footer here with margin auto to get rid of the section and avoid the header warning.